Gruntfile.js

```diff
@@ -95,17 +98,13 @@ module.exports = function( grunt ) {
 			const versionAppend = new Date().toISOString().replace( /\.\d+/, '' ).replace( /-|:/g, '' ) + '-' + commitHash;
 
 			const paths = lsOutput.trim().split( /\n/ ).filter( function( file ) {
-				return ! /^(blocks|\.|bin|([^/]+)+\.(md|json|xml)|Gruntfile\.js|tests|wp-assets|readme\.md|composer\..*|patches|webpack.*|assets\/images\/stories-editor\/.*\.svg|assets\/src|assets\/css\/src|docker-compose\.yml|.*\.config\.js|codecov\.yml|example\.env)/.test( file );
+				return ! /^(blocks|\.|bin|([^/]+)+\.(md|json|xml)|Gruntfile\.js|tests|wp-assets|readme\.md|webpack.*|assets\/images\/stories-editor\/.*\.svg|assets\/src|assets\/css\/src|docker-compose\.yml|.*\.config\.js|codecov\.yml|example\.env)/.test( file );
 			} );
 
-			paths.push( 'vendor/autoload.php' );
+			paths.push( 'composer.*' );
 			paths.push( 'assets/js/*.js' );
 			paths.push( 'assets/js/*.deps.json' );
 			paths.push( 'assets/css/*.css' );
-			paths.push( 'vendor/composer/**' );
-			paths.push( 'vendor/sabberworm/php-css-parser/lib/**' );
-			paths.push( 'vendor/fasterimage/fasterimage/src/**' );
-			paths.push( 'vendor/willwashburn/stream/src/**' );
```

Comment:
There are a lot of files now being included which should not be part of the build (e.g. the `test` files):

```
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/.editorconfig
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/.gitignore
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/CHANGELOG.md
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/LICENSE
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/README.md
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/circle.yml
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/composer.json
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/phpunit.xml
A	wp-content/plugins/amp/vendor/fasterimage/fasterimage/tests/FasterImageTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/.gitignore
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/.travis.yml
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/CHANGELOG.md
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/Doxyfile
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/PATCHES.txt
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/README.md
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/composer.json
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/composer.lock
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/phpunit.xml
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/Sabberworm/CSS/CSSList/AtRuleBlockListTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/Sabberworm/CSS/CSSList/DocumentTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/Sabberworm/CSS/OutputFormatTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/Sabberworm/CSS/ParserTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/Sabberworm/CSS/RuleSet/DeclarationBlockTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/Sabberworm/CSS/RuleSet/LenientParsingTest.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/bootstrap.php
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-calc-no-space-around-minus.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-charset-after-rule.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-charset-in-block.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-empty-grid-linename.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-empty.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-end-token-2.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-end-token.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-fault-tolerance.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/-tobedone.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/1readme.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/2readme.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/atrules.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/calc-nested.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/calc.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/case-insensitivity.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/colortest.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/comments.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/create-shorthands.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/docuwiki.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/empty-grid-linename.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/expand-shorthands.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/functions.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/grid-linename.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/hex-alpha.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/ie-hacks.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/ie.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/important.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/inner-color.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/line-numbers.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/missing-property-value.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/ms-filter.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/namespaces.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/nested.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/slashed.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/specificity.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/trailing-whitespace.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/unicode-range.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/unicode.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/unmatched_braces.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/unopened-close-brackets.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/url.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/values.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/webkit.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/files/whitespace.css
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/phpunit.xml
A	wp-content/plugins/amp/vendor/sabberworm/php-css-parser/tests/quickdump.php
A	wp-content/plugins/amp/vendor/willwashburn/stream/.gitignore
A	wp-content/plugins/amp/vendor/willwashburn/stream/.travis.yml
A	wp-content/plugins/amp/vendor/willwashburn/stream/LICENSE
A	wp-content/plugins/amp/vendor/willwashburn/stream/README.md
A	wp-content/plugins/amp/vendor/willwashburn/stream/composer.json
A	wp-content/plugins/amp/vendor/willwashburn/stream/phpunit.xml
A	wp-content/plugins/amp/vendor/willwashburn/stream/tests/StreamTests.php
```

Can these be excluded? The files add to the ZIP size and they are unused in production.

Reply:
Done in 2a86f5da. This should make it easier to maintain going forward, though it's surely not a good as it can be.